.github/workflows/trigger-tests-on-stable-packages.yml

```diff
@@ -0,0 +1,13 @@
+name: Trigger tests for stable APT packages
+
+on:
+  push:
+    tags:
+      - 'syslog-ng-*'
```

Comment:
Testing the already released version can be useful, but we should probably create a "pre-release"  section next to "stable" and "nightly" so that we can test the release candidate APT packages before the actual release.

Reply:
I am agreed with this too.

Reply:
I think we won't have the capacity to implement this for this release.
@mitzkia What do you think about adding a line for allowing this job to be run manually, so we can start it after uploading the new packages but before announcing the actual release?

Reply:
I am also agreed with this method.